README.md

```diff
@@ -501,6 +501,27 @@ var Status = React.createClass({
 });
 ```
 
+#### Using Reflux.connectFilter
+
+`Reflux.connectFilter` is used in a similar manner to `Reflux.connect`. Use the
+`connectFilter` mixin when you want only a subset of the items in a store. A
+blog written using Reflux would probably have a store with all posts in
+it. For an individual post page, you could use `Reflux.connectFilter` to
+filter the posts to the post that's being viewed.
+
+```javascript
+var PostView = React.createClass({
+    mixins: [Reflux.connectFilter(postStore,"post", function(posts) {
+        posts.filter(function(post) {
+           post.id === this.props.id;
```

Comment:
Heh, that was my problem too.
On Mon, Feb 16, 2015 at 11:25 PM Mikael Brassman <notifications@github.com>
wrote:

> In README.md
> <https://github.com/spoike/refluxjs/pull/222#discussion_r24797357>:
>
> > @@ -501,6 +501,27 @@ var Status = React.createClass({
> >  });
> >  ```
> >
> > +#### Using Reflux.connectFilter
> > +
> > +`Reflux.connectFilter` is used in a similar manner to `Reflux.connect`. Use the
> > +`connectFilter` mixin when you want only a subset of the items in a store. A
> > +blog written using Reflux would probably have a store with all posts in
> > +it. For an individual post page, you could use `Reflux.connectFilter` to
> > +filter the posts to the post that's being viewed.
> > +
> > +```javascript
> > +var PostView = React.createClass({
> > +    mixins: [Reflux.connectFilter(postStore,"post", function(posts) {
> > +        posts.filter(function(post) {
> > +           post.id === this.props.id;
>
> Yup... nice catch.
>
> I've been having too much coffeescript lately to notice. [image: :+1:]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/spoike/refluxjs/pull/222/files#r24797357>.
>